### PR TITLE
Policy Versions

### LINKED ISSUE #123: Policy Versioning
* Show policy version history on the policy details page

### PR DESCRIPTION
**Pull request by pickjasmine**
Shows policy version history in a new tab on the policy page

Closes #123

### CODE REVIEW

File: components/policies/PolicyHistory.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <Loading loading={loading}>
+        {data && (
+          <>
+            {data.map((version, index) => {
+              return (
+                <div key={version.version} className={styles.versionCard}>
+                  <div>
+                    <LabelWithValue
+                      label={"Policy Version"}
+                      value={`${version.version}${
```
Comment (alexashley) [inline components/policies/PolicyHistory.js:47]: Could be nice to display `v1`, `v2`, etc here but I don't feel strongly about it.

**Review by alexashley — COMMENTED** [review]
It doesn't seem like the version history is fetched again after a policy update changes the Rego. Although I do see the new version after a refresh.

File: components/policies/PolicyHistory.js (hunk 1 above)
Comment (pickjasmine) [reply components/policies/PolicyHistory.js:47]: image
Does it look weird to have `Policy Version` still? I'm worried just `v1` won't be enough to connect the dots until you have multiple versions

**Comment by pickjasmine** [comment]
> It doesn't seem like the version history is fetched again after a policy update changes the Rego. Although I do see the new version after a refresh.

I believe this is because of the SWR hook. I can look into forcing the call after creation so you don't have to refresh to see the data

File: components/policies/PolicyHistory.js (hunk 1 above)
Comment (pickjasmine) [reply components/policies/PolicyHistory.js]: I'm gonna go with showing both the label `Policy Version` and `v#`

Comment (alexashley) [reply components/policies/PolicyHistory.js]: I don't think it's too weird, but I see why it might be redundant. I'm fine either way

**Review by alexashley — APPROVED** [review]

**Pull request merged**